models/articlesModels.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection");
+
+exports.fetchArticleByIdFromDb = (id) => {
+  const validId = /[0-9]+/.test(id);
+
+  if (!validId) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }

```

Comment:
I enjoy that you are validating your id but psql will throw an error for us if our datatype does not follow the constraints of the table set up!

Reply:
I've now changed it to handle the correct error :)